Marlin/src/module/temperature.cpp

```diff
@@ -573,12 +591,26 @@ int Temperature::getHeaterPower(const int heater) {
 //
 void Temperature::_temp_error(const int8_t e, PGM_P const serial_msg, PGM_P const lcd_msg) {
   static bool killed = false;
+  /*
   if (IsRunning()) {
     SERIAL_ERROR_START();
     serialprintPGM(serial_msg);
     SERIAL_ERRORPGM(MSG_STOPPED_HEATER);
     if (e >= 0) SERIAL_ERRORLN((int)e); else SERIAL_ERRORLNPGM(MSG_HEATER_BED);
   }
+  */
```

Comment:
No need to comment out old code that's being replaced. Just replace it. We have a version control system to see the changes.